src/routes/relay/entities/relay.dto.entity.ts

```diff
@@ -11,7 +11,7 @@ export class RelayDto {
   data!: string;
 
   @ApiPropertyOptional({
-    type: BigInt,
+    type: String,
```

Comment:
Is `BigInt` not supported?

Reply:
It accepts it type-wise but the request is receiving a `String`, not a `BigInt`.

Reply:
We should consider maybe mapping this to a a `BigInt` before it reaches our domain. Non-numerical values should not work with the relay.

Reply:
Good point. I'll investigate it in a separate PR.